sdks/java/io/amazon-web-services2/src/main/java/org/apache/beam/sdk/io/aws2/kinesis/KinesisIO.java

```diff
@@ -494,17 +495,12 @@ public Read withMaxCapacityPerShard(Integer maxCapacity) {
 
     @Override
     public PCollection<KinesisRecord> expand(PBegin input) {
+      checkNotNull(getAWSClientsProvider(), "AWSClientsProvider cannot be null");
```

Comment:
nit: better to use `checkArgument(...)` in `expand()`

Reply:
👍